setup.py

```diff
@@ -178,7 +178,6 @@
             'hepnames2marc = inspirehep.dojson.hepnames2marc:hepnames2marc',
         ],
         'invenio_base.api_apps': [
-            'inspire_theme = inspirehep.modules.theme:INSPIRETheme',
```

Comment:
Actually, without this you can't run a search on the main page (it fails to find the template), though none of the tests fail :/